lib/topological_inventory/ansible_tower/receptor/api_object.rb

```diff
@@ -28,6 +28,11 @@ def get
         raw_kafka_response(response)
       end
 
+      def post(data)
+        response = send_request(:post, endpoint, :data => data)
+        raw_kafka_response(response)
+      end
```

Comment:
So - as far as I understand this, we're doing a blocking request for it right? That sounds good just want to make sure I'm seeing it right.

Reply:
yes, at this moment it's like availability check